services/files/run_push.go

```diff
@@ -1,18 +1,23 @@
 package files
 
 import (
+	"context"
 	"errors"
 	"fmt"
 	"os"
 	"path/filepath"
 	"strings"
+	"time"
 
 	clierror "github.com/Smartling/smartling-cli/services/helpers/cli_error"
 	globfiles "github.com/Smartling/smartling-cli/services/helpers/glob_files"
 	"github.com/Smartling/smartling-cli/services/helpers/rlog"
 
+	sdkjobs "github.com/Smartling/api-sdk-go/api/jobs"
+	sdktype "github.com/Smartling/api-sdk-go/helpers/file"
 	sdkerror "github.com/Smartling/api-sdk-go/helpers/sm_error"
 	sdkfile "github.com/Smartling/api-sdk-go/helpers/sm_file"
+	"github.com/google/uuid"
```

Comment:
For me, it is an unnecessary dependency. Smartling does not use an uuid generator and instead has its own format (which makes sense and appears to be standard). A simple regex can do the trick.